lib/services/plugins-service.ts

```diff
@@ -20,6 +20,12 @@ export class PluginsService implements IPluginsService {
 
 		return this._identifierToPlugin;
 	}
+	
+	private get oneConfigurationOnlyPlugins(): IStringDictionary {
```

Comment:
Because the idea was to easily add other plugins which we want to have in only one configuration. But in the moment there aren't such plugins, so I can hardcode the id of the LivePatch plugin directly in addPlugin method.
What do you think about this?